medusa/server/api/v2/config.py

```diff
@@ -196,53 +226,94 @@ def data_main():
         section_data['namingForceFolders'] = bool(app.NAMING_FORCE_FOLDERS)
         section_data['subtitles'] = NonEmptyDict()
         section_data['subtitles']['enabled'] = bool(app.USE_SUBTITLES)
+
+        section_data['news'] = NonEmptyDict()
+        section_data['news']['lastRead'] = app.NEWS_LAST_READ
+        section_data['news']['latest'] = app.NEWS_LATEST
+        section_data['news']['unread'] = app.NEWS_UNREAD
+
         section_data['kodi'] = NonEmptyDict()
-        section_data['kodi']['enabled'] = bool(app.USE_KODI and app.KODI_UPDATE_LIBRARY)
+        section_data['kodi']['enabled'] = bool(app.USE_KODI)
+        section_data['kodi']['alwaysOn'] = bool(app.KODI_ALWAYS_ON)
+        section_data['kodi']['notify'] = NonEmptyDict()
+        section_data['kodi']['notify']['snatch'] = bool(app.KODI_NOTIFY_ONSNATCH)
+        section_data['kodi']['notify']['download'] = bool(app.KODI_NOTIFY_ONDOWNLOAD)
+        section_data['kodi']['notify']['subtitleDownload'] = bool(app.KODI_NOTIFY_ONSUBTITLEDOWNLOAD)
+        section_data['kodi']['update'] = NonEmptyDict()
+        section_data['kodi']['update']['library'] = bool(app.KODI_UPDATE_LIBRARY)
+        section_data['kodi']['update']['full'] = bool(app.KODI_UPDATE_FULL)
+        section_data['kodi']['update']['onlyFirst'] = bool(app.KODI_UPDATE_ONLYFIRST)
+        section_data['kodi']['host'] = app.KODI_HOST
+        section_data['kodi']['username'] = app.KODI_USERNAME
+        # section_data['kodi']['password'] = app.KODI_PASSWORD
```

Comment:
Imo we can leave these out.
Can you remove it, in stead of commenting?

Reply:
I don't mind leaving out the passwords. But then please remove the commented lines.

Reply:
I left the comments I could find out if we needed them removed or not.